packages/electron/prepare-build.js

```diff
@@ -33,11 +33,13 @@ async function main() {
     const buildConfig = {
         appId,
         buildVersion,
+        artifactName: "${name}_${version}_${os}_electron_${arch}.${ext}",
         directories: {
             app: "app",
             buildResources: "build",
         },
         mac: {
+            artifactName: "${name}_${version}_macos_electron_${arch}.${ext}",
```

Comment:
`${os}` for `mac` is `mac` and not `macos`, which we're using in the assets, thus the need for this platform-specific `artifactName`.